vagrant_box_defaults.rb

```diff
@@ -6,3 +6,5 @@
 $SERVER_VERSION= "167"
 $NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
 $NETNEXT_SERVER_VERSION= "45"
+@v419_SERVER_BOX= "cilium/ubuntu-4-19"
```

Comment:
What's the difference between the initialization of a variable with `$` and `@`?

Reply:
`@` allows to get variable value by string value with `instance_variable_get`. Variables that don't start with `@` are not allowed to be retrieved like that #justrubythings

Reply:
Prefixing a variable with actually makes it [global](https://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/globalvars.html). I don't think there is a reason for using globals in Vagrantfiles, it just seems to have become somewhat of a tradition, probably due to some funny historical detail...

Reply:
I mean, people usually copy a Vagrantfile example, and I think there are lots of examples with global vars. Here is one I used hack on: https://github.com/errordeveloper/weave-demos/blob/master/coreos-vagrant/Vagrantfile.